docs/source/serving/update.rst

```diff
@@ -1,42 +1,56 @@
 .. _serve-update:
 
-Update Your Service
-===========
+Updating a Service
+==================
+
+SkyServe supports *updating* a deployed service, which can be used to change:
+
+* Replica code (e.g., ``run``/``setup``; useful for debugging)
+* Replica resource spec (e.g., number of replicas; accelerator/instance used)
+* Service spec (e.g., autoscaling spec in ``replica_policy``)
 
-SkyServe supports update for your services. Use ``sky serve update`` to update an existing service:
+During an update, the service endpoint will remain the same and the service will
+remain accessible with no downtime.
+
+Use ``sky serve update`` to update an existing service:
 
 .. code-block:: console
 
     $ sky serve update service-name new_service.yaml
 
-SkyServe will launch new replicas described by ``new_service.yaml``. When the number of new replicas reaches the minimum number of replicas (``min_replicas``) required for the service, SkyServe will scale down old replicas to save cost. During the whole process, the service is still accessible to users. SkyServe allows users to update ``replica_policy`` parameters, such as ``target_qps_per_replica``. SkyServe also allows users to update ``resources`` parameters, such as ``cpu`` and ``memory``, so that new replicas can be launched on VMs of different types.   
+SkyServe will launch new replicas described by ``new_service.yaml`` with the following behavior:
+
+* An update is initiated, and traffic will still be redirected to existing (old) replicas.
```

Comment:
```suggestion
* An update is initiated, and traffic will be redirected to existing (old) replicas as new replicas are provisioning
```

Reply:
Do you mean "since new replicas are still provisioning we have to send traffic to old", or "..while new replicas are being provisioning"? If the latter, it should be covered by the following 2 bullets.

Reply:
The former. I think the current one looks fine. Maybe remove  'still'

Reply:
Reworded a bit to "traffic will continue to be redirected to existing (old) replicas." which seems better.